src/test/java/api/requests/RequestsAPILoanRenewalTests.java

```diff
@@ -171,4 +192,46 @@ public void allowOverrideRenewalLoanByBarcodeWhenFirstRequestInQueueIsHold() thr
 
     assertThat(response.getJson().getString("action"), is("renewedThroughOverride"));
   }
+
+  @Test
+  public void forbidRenewalOverrideWhenFirstRequestIsNotRecall()
+    throws MalformedURLException,
+    InterruptedException,
+    TimeoutException,
+    ExecutionException {
+
+    final InventoryItemResource smallAngryPlanet = itemsFixture.basedUponSmallAngryPlanet();
+    final IndividualResource rebecca = usersFixture.rebecca();
+
+    DateTime loanDueDate = new DateTime(
+      2018, APRIL, 21,
+      11, 21, 43
+    );
+
+    loansFixture.checkOutByBarcode(smallAngryPlanet, rebecca, loanDueDate);
+
+    useLoanPolicyAsFallback(
+      loanPoliciesFixture.canCirculateRolling().getId(),
+      requestPoliciesFixture.allowAllRequestPolicy().getId(),
+      noticePoliciesFixture.activeNotice().getId()
+    );
+
+    requestsClient.create(new RequestBuilder()
```

Comment:
Why is this using the requestsClient rather than the fixture?

Reply:
Copied it from another test in this module. I didn't noticed that, so I'm going to replace it with fixture call.

Reply:
Thank you

Reply:
Done